### PR TITLE
add sip config files

### PR DESCRIPTION
**Pull request by TECH7Fox**
A start to allow custom extensions.

### CODE REVIEW

File: Asterisk/run.sh [hunk 1]
```diff
@@ -51,10 +51,20 @@ bashio::var.json \
...
 
+# Make a loop for these
+
 tempio \
     -template /usr/share/tempio/rtp.conf.gtpl \
     -out /etc/asterisk/rtp.conf
 
+tempio \
+    -template /usr/share/tempio/sip_custom.conf.gtpl \
+    -out /etc/asterisk/sip_custom.conf
```
Comment (felipecrs) [inline Asterisk/run.sh:62]: Unless you plan to add templating capabilities, I think we don't need tempio for such custom files.

File: Asterisk/run.sh [hunk 2]
```diff
@@ -51,10 +51,20 @@ bashio::var.json \
...
+
+tempio \
+    -template /usr/share/tempio/sip_default.conf.gtpl \
+    -out /etc/asterisk/sip_default.conf
```
Comment (felipecrs) [inline Asterisk/run.sh:66]: I think you forgot to supply the configuration to tempio here.

File: Asterisk/run.sh (hunk 1 above)
Comment (TECH7Fox) [reply Asterisk/run.sh:76]: true, we have more files like that. Maybe we should move them to rootfs/etc/asterisk?

File: Asterisk/run.sh (hunk 2 above)
Comment (TECH7Fox) [reply Asterisk/run.sh]: yes, I swapped it with `sip.conf`.

File: Asterisk/run.sh (hunk 1 above)
Comment (TECH7Fox) [reply Asterisk/run.sh]: Moved to `/etc/asterisk` in https://github.com/TECH7Fox/Asterisk-add-on/pull/27/commits/2900e08f9f4ee83bc4694b16c07ed6c9a04c4793

File: asterisk/rootfs/etc/cont-init.d/asterisk.sh [hunk 3]
```diff
@@ -27,7 +27,6 @@ fi
 readonly target_certfile="/etc/asterisk/keys/fullchain.pem"
 readonly target_keyfile="/etc/asterisk/keys/privkey.pem"
 
-mkdir -p /etc/asterisk/keys
```
Comment (felipecrs) [inline asterisk/rootfs/etc/cont-init.d/asterisk.sh:30]: I think this is needed

Comment (TECH7Fox) [reply asterisk/rootfs/etc/cont-init.d/asterisk.sh:30]: Fixed in https://github.com/TECH7Fox/Asterisk-add-on/pull/27/commits/16e22f4d6c761f6c2c19eb603a3ef80c2ff4f692

File: asterisk/rootfs/etc/cont-init.d/asterisk.sh [hunk 4]
```diff
@@ -72,12 +65,10 @@ bashio::var.json \
...
-        -template /usr/share/tempio/sip.conf.gtpl \
-        -out /etc/asterisk/sip.conf
+        -template /usr/share/tempio/sip_default.conf.gtpl \
+        -out /config/asterisk/sip_default.conf
 
-if ! bashio::fs.file_exists '/config/asterisk/sip.conf'; then
-    cp -a /etc/asterisk/. /config/asterisk/ ||
-        bashio::exit.nok 'Failed to make sample configs'
-fi
+sed -i 's/noload => chan_sip.so/;noload => chan_sip.so/' /etc/asterisk/modules.conf >/dev/null
```
[the remark this answers is not in the document]
Comment (TECH7Fox) [reply asterisk/rootfs/etc/cont-init.d/asterisk.sh:71]: Was thinking about that too. I will add that.

[the remark this answers is not in the document]
Comment (TECH7Fox) [reply asterisk/rootfs/etc/cont-init.d/asterisk.sh]: Done

**Pull request merged**